plugins/repoquery.py

```diff
@@ -97,12 +100,16 @@ def parse_arguments(args):
                         help=_('show only results there provides REQ'))
     parser.add_argument('--whatrequires', metavar='REQ',
                         help=_('show only results there requires REQ'))
+    parser.add_argument("--alldeps", action="store_true",
+                        help="shows results that requires package provides and files")
     parser.add_argument('--querytags', action='store_true',
                         help=_('show available tags to use with '
                                '--queryformat'))
     parser.add_argument('--resolve', action='store_true',
                         help=_('resolve capabilities to originating package(s)')
-                       )
+                        )
+    parser.add_argument("--tree", action="store_true", 
+                        help="For the given packages print a tree of the packages.")
```

Comment:
What about adding this to the `outform`? I think that this is just another type of output. What do you think?